torchvision/ops/misc.py

```diff
@@ -126,6 +126,7 @@ class FrozenBatchNorm2d(torch.nn.Module):
 
     def __init__(self, n):
         super(FrozenBatchNorm2d, self).__init__()
+        self.num_features = n
```

Comment:
Is this for keeping compatibility with `BatchNorm` interface? Because this value can be inferred from `self.weight`, right?

Reply:
Good point!